components/BottomNavigation/src/private/MDCBottomNavigationCell.h

```diff
@@ -0,0 +1,35 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+
+@interface MDCBottomNavigationCell : UIView
```

Comment:
If you are not inheriting from UICollectionViewCell, I wouldn't call this MDCXYZCell.
MDCBottomNavigationView?

Should this inherit from UIControl instead of UIView since it handles user interaction?

Naming is hard.

Reply:
Renamed this class to MDCBottomNavigationItemView. However, I think this should remain a UIView rather than a UIControl since UIControlStates don't match what's in the design spec.